tests/contracts-solidity.spec.ts

```diff
@@ -63,12 +66,7 @@ describe("Solang Smart Contracts", () => {
   test("Raw Flipper contract", async (done): Promise<void> => {
     // The next two lines are a not so pretty workaround until the new metadata format has been fully implemented
     const metadata = require("../contracts/solidity/flipper/flipper.json");
-    const selector = u8aToHex(new Uint8Array(
-      JSON.parse(metadata.contract.constructors[0].selector)
-    ));
-    const flipAction = u8aToHex(new Uint8Array(
-     JSON.parse(metadata.contract.messages[0].selector)
-    ));
+    const abi = new Abi(api.registry, metadata);
```

Comment:
Ah, cool, that's working again? It used to be broken so I haven't tried it in a while!

Reply:
It seems to be working. I have no idea how to decode return values though.